galley/widgets.py

```diff
@@ -354,7 +353,7 @@ def push(self, frame):
         self.frames.append(frame)
 
         self._apply(1)
-        # print "START CONTEXT", frame.node, self.origin
+        # print('START CONTEXT', frame.node, self.origin)
```

Comment:
For this, and all the others, since you're changing these lines anyway and they're commented out, let's either remove them or make them `log.debug` which would require importing logging.

Reply:
+1 OK; I will make them `log.debug`